### PR TITLE
feat(api): add sentry dsn in web config

### PR DESCRIPTION
**Pull request by almostinf**
# Add sentry dsn in web config

Also refactor `getWebConfig` route

### CODE REVIEW

**Review by Tetrergeru — APPROVED** [review]

File: api/config.go [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+
+// FeatureFlags is struct to manage feature flags.
+type FeatureFlags struct {
+	IsPlottingDefaultOn              bool `json:"isPlottingDefaultOn" example:"false"`
+	IsPlottingAvailable              bool `json:"isPlottingAvailable" example:"true"`
+	IsSubscriptionToAllTagsAvailable bool `json:"isSubscriptionToAllTagsAvailable" example:"false"`
+	IsReadonlyEnabled                bool `json:"isReadonlyEnabled" example:"false"`
+}
+
+type Sentry struct {
```
Comment (kissken) [inline api/config.go:25]: godoc 

not necessary

**Review by kissken — APPROVED** [review]

File: api/config.go [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+	DSN string `json:"dsn" example:"https://secret@sentry.host"`
```
Comment (kissken) [inline api/config.go:26]: omitempty?

Comment (almostinf) [reply api/config.go]: fixed

File: api/config.go (hunk 1 above)
Comment (almostinf) [reply api/config.go:26]: fixed

**Review by kissken — APPROVED** [review]

**Review by Tetrergeru — APPROVED** [review]

**Pull request merged**